contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -160,7 +161,6 @@ contract RyskStrategy is BaseStrategy {
         onlyManager
     {
         if (_amount == 0) revert StrategyAmountZero();
-        if (_amount > investedAssets()) revert StrategyNotEnoughShares();
```

Comment:
why is this no longer relevant?

Reply:
because we are doing the withdrawals from the pool in shares, not in the underlying amount